modules/api/src/main/java/com/intuit/wasabi/api/ApplicationsResource.java

```diff
@@ -395,19 +396,13 @@ public Response getAllExperimentTags(
             @ApiParam(value = EXAMPLE_AUTHORIZATION_HEADER, required = true)
             final String authorizationHeader) {
         try {
-
-            // get the for the user visible Applications
-            List<Application.Name> allApplications = experiments.getApplications();
-            Set<Application.Name> allowed = new HashSet<>();
             UserInfo.Username userName = authorization.getUser(authorizationHeader);
+            Set<Application.Name> allowed = new HashSet<>();
```

Comment:
@AndreaSuckro Thanks for making these changes quickly! You could possibly rename this variable to `authorizedApplications` for better readability.